tests/e2e/specs/dashboard/screens.test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Dashboard screens test.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { enableFeature, setupSiteKit } from '../../utils';
+
+describe( 'dashboard screens', () => {
```

Comment:
As mentioned before, let's make these non-"dashboard"-specific.

Reply:
Thanks @aaemnnosttv this has been resolved in a recent commit, I've simply made it less focussed on dashboard and more just about screens.